lisp/casual-bookmarks.el

```diff
@@ -61,7 +61,8 @@
     ("r" "Rename…" bookmark-bmenu-rename :transient nil)
     ("R" "Relocate…" bookmark-bmenu-relocate :transient nil)
     ("x" "Delete" bookmark-bmenu-execute-deletions :transient t)
-    ("w" "Show location" bookmark-bmenu-locate :transient t)]
+    ("w" "Show location" bookmark-bmenu-locate :transient t)
+    ("+" "Add bookmark" casual-set-bookmark-in-buffer :transient t)]
```

Comment:
Please squash this commit.

Reply:
Should be squashed now. Sorry about that.